Pipfile

```diff
@@ -8,6 +8,7 @@ certifi = "==2018.10.15"
 "connexion[swagger-ui]" = "~=2.2"
 crate = "~=0.22"
 flask = "~=1.1.0"
+flask_cors = "~=3.0.8"
```

Comment:
is flask cors `3.0.8` compatible with the rest of the web stack? i.e. flask `1.1.0`, connexion `2.2`, ...